content/en/continuous_integration/tests/javascript.md

```diff
@@ -415,6 +415,10 @@ If you are running tests in non-supported CI providers or with no `.git` folder,
 : Commit committer date in ISO 8601 format.<br/>
 **Example**: `2021-03-12T16:00:28Z`
 
+## Git metadata upload
+
+From `dd-trace>=3.15.0` and `dd-trace>=2.28.0`, CI Visibility automatically uploads git metadata information (commit history). This metadata contains file names but no file contents. If you want to opt out of this behavior, you can do so by setting `DD_CIVISIBILITY_GIT_UPLOAD_ENABLED` to `false`. However, this is not recommended, as features like Intelligent Test Runner and others do not work without it.
```

Comment:
```suggestion
From `dd-trace>=3.15.0` and `dd-trace>=2.28.0`, CI Visibility automatically uploads git metadata information (commit history). This metadata contains file names but no file content. If you want to opt out of this behavior, you can do so by setting `DD_CIVISIBILITY_GIT_UPLOAD_ENABLED` to `false`. However, this is not recommended, as features like Intelligent Test Runner and others do not work without it.
```

Reply:
I think "file contents" is [more common](https://books.google.com/ngrams/graph?content=file+content%2Cfile+contents&year_start=2010&year_end=2019&corpus=en-2019&smoothing=0&case_insensitive=true)?

Reply:
Thanks for pointing that. I think both could work, but ok for the more commonly used one :)